### PR TITLE
Standardize a pull requests template

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

I believe we should enforce how our pull requests shall look, and be more consistent of what they include. For example, it's easy to forget to add images of the result or to label the PR properly. A [pull request template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates) should hopefully help a long way.

Curious to what you guys figure of this, and I'm all ears. If desired, we may add a template for issues as well.

I wrote a `CONTRIBUTING.md` as well, which I believe will make it easier for newcomers.

# Result

You're sort of looking at it.

# Testing

- [ ] I have thoroughly tested my changes.

I have **not** had the opportunity to test this, as I believe the `PULL_REQUEST_TEMPLATE.md` must exist in the repository **before** the pull request is opened. Crossing fingers 🤞🏼...

### CODE REVIEW

**Comment by ivarnakken** [comment]
Should probably add a `CONTRIBUTING.md` file to our repository as well, and link to it in the template. Will do tomorrow.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Description
+
+Please include a summary of the changes. Also, please include relevant motivation and context. Remember to label your pull request properly, of which the type of change should be included; **bug-fix**, **new-feature**, **enhancement**, **docs** or **breaking-change**.
+
+# Result
+
+If you've made visual changes, please include **before** and **after** images, preferably with a description.
```
Comment (juniwbjerde) [inline .github/pull_request_template.md:7]: Can we add a quick line about not including real user information in the images? I know it's quite obvious for most Webkommers now, but I remember doing so when I was new and nobody had told me not to... It wouldn't hurt anybody, but could help someone with a friendly reminder:)

**Review by juniwbjerde — APPROVED** [review]
Love this! It's very helpful and will probably up the quality of our PRs!

**Comment by ivarnakken** [comment]
A `CONTRIBUTING.md` has now been written. Please review.

**Review by ingraso — APPROVED** [review]
Looks great! Very nice with the introductory links and good explanations :tada:

**Pull request merged**